pkg/api/types.go

```diff
@@ -256,6 +258,19 @@ type SecretVolumeSource struct {
 	Target ObjectReference `json:"target"`
 }
 
+// NFS represents and NFS Mount that lasts the lifetime of a pod
+type NFSVolumeSource struct {
+	// Server is the hostname or ip address of the NFS server
+	Server string `json:"server"`
+
+	// SourcePath is the exported NFS share
+	SourcePath string `json:"sourcePath"`
```

Comment:
clearer, thanks.

nit: sourcePath vs path - does "source" add information enough to justify itself?